docs/data-sources/er_flow_logs.md

```diff
@@ -0,0 +1,86 @@
+---
+subcategory: "Enterprise Router (ER)"
+---
+
+# huaweicloud_er_flow_logs
+
+Use this data source to get the list of flow logs.
+
+## Example Usage
+
+```hcl
+variable "resource_id" {}
+
+data "huaweicloud_er_flow_logs" "test" {
+  resource_id = var.resource_id
```

Comment:
the required `instance_id` param is missing.

Reply:
done